app/views/view-application.njk

```diff
@@ -27,6 +28,7 @@
       {% if withdrawConfirmationForm %}
         {{ viewWithdrawConfirmationForm(applicationId, page, crumb) }}
       {% endif %}
+      {{ viewClaimConfirmationForm(applicationId, page, crumb) }}
```

Comment:
Does this need a check around it so that the claim confirmation is only shown to specific users and when the claim is in a certain state i.e. In Check?